lib/ramble/ramble/workspace/workspace.py

```diff
@@ -62,7 +62,10 @@
 _active_workspace = None
 
 #: Path where workspaces are stored
-workspace_path = os.path.join(ramble.paths.var_path, 'workspaces')
+workspace_path = ramble.config.get('config:workspace_dirs')
+if not workspace_path:
+    workspace_path = os.path.join(ramble.paths.var_path, 'workspaces')
```

Comment:
We should revert to building a path from `ramble.paths.var_path` if `config:workspace_dirs` is not present. Instead, we should only rely on it being defined in `config:workspace_dirs`. And then some commands might need to be updated to properly handle the case where the directory might not exist.

Reply:
Did you mean "We should *not* revert to building a path ..."?

IIUC, you mean that since ramble.config.get('config:workspace_dirs') is now getting the default location unless redefined by the user, we don't have to assemble the default location here also.

Currently, there is no way to handle the case where there is no workspaces under var_path, say if someone has deleted the directory. So do you want to add new functionality that will handle that also, or at least report an error?

Reply:
Yeah, sorry. We should *not* revert to building a path.

Yes, basically, we should make the only path for getting the workspace_dir through a config, and not have a hardcoded path in `ramble.paths` in addition to a config set one.

I don't know that we need to throw an error. If the directory `config:workspace_dirs` points to doesn't exist, there wouldn't be any workspaces in it. The only place that an error would occur is if ramble tried to create this directory, and didn't have permissions. But, I think that's handled when trying to create a workspace the first time anyway.

It's worth testing at least.